README.md

```diff
@@ -129,7 +129,7 @@ def parse(self, response):
     ...
 ```
 
-#### Using css instead of xpath for classes
+#### Using xpath css instead of css for classes
```

Comment:
Looks like the intent of the heading wasn't clear - does this make more sense?
Primarily this section is about how css is better than xpath for classes. Secondly it tries to confirm that yes, both are options.

```suggestion
#### CSS lookup is preferred over XPath for HTML classes
```